c++/CMakeLists.txt

```diff
@@ -82,11 +82,40 @@ else()
 endif()
 
 # Source =======================================================================
-
+include(CapnProtoMacros)
 add_subdirectory(src)
 
 # Install ======================================================================
 
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+  "${CMAKE_CURRENT_BINARY_DIR}/CapnProtoConfigVersion.cmake"
+  VERSION ${VERSION}
+  COMPATIBILITY AnyNewerVersion
+)
+set(ConfigPackageLocation ${LIB_INSTALL_DIR}/cmake/CapnProto)
+
+configure_package_config_file(cmake/CapnProtoConfig.cmake.in
+  ${CMAKE_CURRENT_BINARY_DIR}/CapnProtoConfig.cmake
+  INSTALL_DESTINATION ${ConfigPackageLocation}
+  PATH_VARS INCLUDE_INSTALL_DIR
+)
+export(EXPORT CapnProtoTargets
+  FILE "${CMAKE_CURRENT_BINARY_DIR}/CapnProtoTargets.cmake"
+  NAMESPACE CapnProto::
+)
+install(EXPORT CapnProtoTargets
+  FILE CapnProtoTargets.cmake
+  NAMESPACE CapnProto::
+  DESTINATION ${ConfigPackageLocation}
+)
+install(FILES
+  cmake/CapnProtoMacros.cmake
+  ${CMAKE_CURRENT_BINARY_DIR}/CapnProtoConfig.cmake
+  ${CMAKE_CURRENT_BINARY_DIR}/CapnProtoConfigVersion.cmake
+  DESTINATION ${ConfigPackageLocation} COMPONENT Devel
```

Comment:
I'm not very familiar with it, but from a quick skim of the documentation, it seems to be tied to CPack? I can't see a method for excluding a component from a "make install" without building a CPack installer.